app/src/main/java/com/example/eric/memberquiz/GameActivity.java

```diff
@@ -0,0 +1,306 @@
+    Button button1;
+    Button button2;
+    Button button3;
+    Button button4;
+    Button endButton;
+    TextView score;
+    TextView countdownText;
+    SharedPreferences pref;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_game);
+
+        String[] members = {"Jessica Cherny", "Kevin Jiang", "Jared Gutierrez", "Kristin Ho", "Christine Munar", "Mudit Mittal", "Richard Hu", "Shaan Appel", "Edward Liu", "Wilbur Shi", "Young Lin", "Abhinav Koppu", "Abhishek Mangla", "Akkshay Khoslaa", "Andy Wang", "Aneesh Jindal", "Anisha Salunkhe", "Ashwin Vaidyanathan", "Cody Hsieh", "Justin Kim", "Krishnan Rajiyah", "Lisa Lee", "Peter Schafhalter", "Sahil Lamba", "Sirjan Kafle", "Tarun Khasnavis", "Billy Lu", "Aayush Tyagi", "Ben Goldberg", "Candice Ye", "Eliot Han", "Emaan Hariri", "Jessica Chen", "Katharine Jiang", "Kedar Thakkar", "Leon Kwak", "Mohit Katyal", "Rochelle Shen", "Sayan Paul", "Sharie Wang", "Shreya Reddy", "Shubham Goenka", "Victor Sun", "Vidya Ravikumar"};
+        ArrayList<String>image_names = new ArrayList<>();
+        Field[] drawables = com.example.eric.memberquiz.R.drawable.class.getFields();
+        for (Field f : drawables) {
+            try {
+                String name = f.getName();
+                if (!name.contains("_") && !name.contains("$") && !name.equals("serialVersionUID")){
+                    System.out.println(name);
+                    image_names.add(name);
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+        System.out.println(image_names.size());
+
+        // Get random image
+        Random rand = new Random();
+        int rand_img_num = rand.nextInt(44);
+        String rand_img = image_names.get(rand_img_num);
+
+        // Setting the random image
+        ImageView image = (ImageView) findViewById(R.id.imageView);
+        int id = getApplicationContext().getResources().getIdentifier("drawable/"+rand_img, null, getApplicationContext().getPackageName());
+        image.setImageResource(id);
+
+        button1 = (Button) findViewById(R.id.button1);
+        button2 = (Button) findViewById(R.id.button2);
+        button3 = (Button) findViewById(R.id.button3);
+        button4 = (Button) findViewById(R.id.button4);
+        endButton = (Button) findViewById(R.id.endButton);
+        score = (TextView) findViewById(R.id.score);
+        countdownText = (TextView) findViewById(R.id.countdownText);
+        pref = getApplicationContext().getSharedPreferences("MyPref", 0);
+
+        int new_score = pref.getInt("score", 0);
+        score.setText(Integer.toString(new_score));
+
+        Button[]answer_buttons = new Button[4];
+        answer_buttons[0] = button1;
+        answer_buttons[1] = button2;
+        answer_buttons[2] = button3;
+        answer_buttons[3] = button4;
+
+        //Clear All Button Text
```

Comment:
Good practice is that if you need to comment a block of code, it should be in a separate method